inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -920,10 +920,12 @@ std::shared_ptr<ICNNNetwork> V10Parser::parse(
     XmlDeserializer visitor(root, weights, opsets, variables);
     bool use_framework_node{false};
     for (const auto & ext : _exts) {
-        InferenceEngine::Version * version = new InferenceEngine::Version();
-        ext->GetVersion(const_cast<const Version *&>(version));
-        if (version->description && strcmp(version->description, "framework_node_ext") == 0) {
+        InferenceEngine::Version version;
+        InferenceEngine::Version* version_ptr = &version;
+        ext->GetVersion(const_cast<const Version *&>(version_ptr));
```

Comment:
I'm not sure if such usage of 'GetVersion' is correct. Function returns "reference to pointer", so the actual pointer will point to another address after return. So the usage may be like this:
`const InferenceEngine::Version *version = nullptr; //!
ext->GetVersion(version);
if (version && version->description && strcmp(...)) {...}
`

Reply:
agree, fixed.